tests/crypto/mbedtls/prj.conf

```diff
@@ -6,3 +6,4 @@ CONFIG_MBEDTLS_TEST=y
 CONFIG_ZTEST=y
 CONFIG_TEST_USERSPACE=y
 CONFIG_MINIMAL_LIBC_RAND=y
+CONFIG_NOCACHE_MEMORY=y
```

Comment:
this change doesn't seem relevant to your PR

Reply:
I was trying to fix the CI/CD issues (The spi loopback CI unit test was failing because there was no non-cacheable memory in which to locate the TCDPool). I've pushed a new change that I _think_ should fix this without these kconfig options though.